frontend/encore/vuejs.rst

```diff
@@ -34,6 +35,70 @@ also configure the `vue-loader options`_ via a callback:
         };
     });
 
+Building your first component
+-----------------------------
+
+Once Vue is installed and webpack.config.js has been updated, let's build our first component and display a simple ``Hello World from Vue !``, in order to ease the process, we gonna use the ``.vue`` files, let's start by adding a ``main.js`` file inside the assets folder : 
```

Comment:
Can you please add line breaks? As a standard, we break our lines after the first word that crosses the 72nd character. Or, you can just make your lines approximately as long as the rest of this file :)

Also, `webpack.config.js` near the beginning needs the code "ticks" around it.

And there is `folder :` => `folder:`

And it's sometimes a bit difficult, but we don't use `I` or `we` in the docs. Can you re-word a little bit?